ui/raidboss/data/03-hw/raid/a3n.txt

```diff
@@ -9,59 +9,59 @@ hideall "--sync--"
 
 # Condensate Demineralizer #9 will be sealed off
 0.0 "--sync--" sync / 29:[^:]*:7DC:[^:]*:66C:/ window 10000,10000
-7.5 "Fluid Swing" sync /:Living Liquid:12EE:/
-11.6 "Splash x3" duration 3 #sync /:Living Liquid:12EF:/
-19.9 "Protean Wave" sync /:Living Liquid:12F0:/
-23.5 "Fluid Swing" sync /:Living Liquid:12EE:/
-27.6 "Splash x3" duration 3 #sync /:Living Liquid:12EF:/
-32.9 "Fluid Swing" sync /:Living Liquid:12EE:/
-39.0 "Sluice" sync /:Living Liquid:12F2:/
-42.0 "--sync--" sync /:Living Liquid:12F3:/
-44.0 "--sync--" sync /:Living Liquid:12F4:/
-47.2 "Fluid Swing" sync /:Living Liquid:12EE:/
-53.3 "Splash x3" duration 3 #sync /:Living Liquid:12EF:/
+7.5 "Fluid Swing" sync / 1[56]:[^:]*:Living Liquid:12EE:/
+11.6 "Splash x3" duration 3 #sync / 1[56]:[^:]*:Living Liquid:12EF:/
+19.9 "Protean Wave" sync / 1[56]:[^:]*:Living Liquid:12F0:/
+23.5 "Fluid Swing" sync / 1[56]:[^:]*:Living Liquid:12EE:/
+27.6 "Splash x3" duration 3 #sync / 1[56]:[^:]*:Living Liquid:12EF:/
+32.9 "Fluid Swing" sync / 1[56]:[^:]*:Living Liquid:12EE:/
+39.0 "Sluice" sync / 1[56]:[^:]*:Living Liquid:12F2:/
+42.0 "--sync--" sync / 1[56]:[^:]*:Living Liquid:12F3:/
+44.0 "--sync--" sync / 1[56]:[^:]*:Living Liquid:12F4:/
+47.2 "Fluid Swing" sync / 1[56]:[^:]*:Living Liquid:12EE:/
+53.3 "Splash x3" duration 3 #sync / 1[56]:[^:]*:Living Liquid:12EF:/
```

Comment:
I'm genuinely surprised these weren't correct at the time. I updated `make_timeline` something like two weeks before this was merged!